src/lwc/alertBanner/alertBanner.html

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
Assuming this is now tracked under "utilAlertBanner" I think we need to remove this LWC ("alertBanner") from the branch!

Reply:
done!